source/LoadInsight.py

```diff
@@ -0,0 +1,125 @@
+import pandas as pd
+
+def main() :
+	"""Dummy main to test basic pipeline functionality"""
+	input1 = DataObject()
+	output1 = DataObject()
+	transform = Transformation(inputs=[input1],outputs=[output1],function=nop)
+	p1 = Pipeline()
+	p1.add(transform)
+	p1.run()
+
+def nop(x):
+	return(x)
+
+def rand(x,N,M):
+	import numpy as np
+	pd.DataFrame(np.random.randn(N,M))
+
+class DataObject:
+	"""DataObject hold all data artifacts as pandas DataFrames 
+	   and associated storage specifications.
+	"""
+	local_storage = {"path":""}
+	remote_storage = {} # TODO: from infrastucture
+
+	def __init__(self,
+			name="NULL",
```

Comment:
We should leverage python's `None` instead of hardcoding a semantically brittle `"NULL"`